tensorboard/uploader/auth.py

```diff
@@ -54,14 +63,30 @@
     "token_uri": "https://oauth2.googleapis.com/token",
     "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
     "client_secret": "pOyAuU2yq2arsM98Bw5hwYtr",
-    "redirect_uris": [
-      "urn:ietf:wg:oauth:2.0:oob",
-      "http://localhost"
-    ]
+    "redirect_uris": ["http://localhost"]
   }
 }
 """
 
+# The config for authenticating using a separate device with a browser, to be
+# used when the environment where this uploader is running does not have a
+# browser available.
+#
+# The client "secret" is considered public, as it's distributed to the client
+# devices where this runs. See:
+# https://developers.google.com/identity/protocols/oauth2/limited-input-device
+#
+# Also see comment in previous constant.
```

Comment:
Done. Rephrased a bit differently, but it's the same idea. Thanks for the suggestion.

Reply:
Actually, upon second inspection, this wasn't accurate. Ended up consolidating the configs into one variable. Changed it to be directly a dictionary instead of a json that we have to parse before using it. Let me know if there are any concerns with that.